src/common/models/allof.go

```diff
@@ -15,16 +16,14 @@ func GetFromAllOf[Inner any, AllOf any](allOf AllOf) *Inner {
 	bin, err := json.Marshal(allOf)
 
 	if err != nil {
-		// TODO replace with log
-		// fmt.Printf("Error when extracting JSON from (%s): %v\n", typename[AllOf](), err)
+		log.Err(err).Msg(fmt.Sprintf("Error when extracting JSON from (%s)\n", typename[AllOf]()))
```

Comment:
Apparently we can do `Msgf` which lets you format the string, making `fmt.Sprintf` redundant - I'd update this to use that

Reply:
done thanks :D was not aware of it

Reply:
me neither 😄